doc/languages-frameworks/rust.section.md

```diff
@@ -686,7 +686,7 @@ $ cargo build
 $ cargo test
 ```
 
-## Using community maintained Rust toolchains {#using-community-maintained-rust-toolchains}
+#### Using community maintained Rust toolchains {#using-community-maintained-rust-toolchains}
```

Comment:
This is not part of "Setting up `nix-shell`", it should belong to its own section where "Using Rust nightly with `nix-shell`" and "Using Rust nightly in a derivation with `buildRustPackage`" are its subsections

Reply:
Thanks for catching that; I rebased incorrectly.